### ISSUE TITLE
The ability to bookmark is getting blocked

### ISSUE THREAD

**Issue by FlappyJay** [H4lyc0n/Goguardian-killer#1]
Some districts are blocking access to the ability to bookmark pages. This is obviously a problem with this, as it makes this bookmarklet no longer work.

**Comment by vuduue** [comment]
yeah, it works on unmanaged computers or Chromebooks with older versions (older versions don't let IT disable bookmarklets), so if you don't have one of those, you're screwed. those are pretty much the only two ways I know you can access this.
